CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 
 ### Fixed
+* [ALFREDAPI-463](https://xenitsupport.jira.com/browse/ALFREDAPI-463): Fix quotation marks in searches being improperly escaped
```

Comment:
Like I said in our daily this morning: make it clear some searches can break here:
`Fix quotation marks in searches being improperly escaped. Search queries can now be escaped properly. E.g. \"Compas Format\" instead of \\\"Compas Format\\\" as was needed previously. `